test/client/fixtures/is-focusable-test.js

```diff
@@ -0,0 +1,98 @@
+var domUtils      = hammerhead.utils.dom;
+var browserUtils  = hammerhead.utils.browser;
+var iframeSandbox = hammerhead.sandbox.iframe;
+
+
+var TEST_ELEMENT_CLASS      = 'testElement';
+var focusedElements         = [];
+var expectedFocusedElements = [];
+var iframe                  = null;
+
+var createElements = function () {
```

Comment:
I propose to put all markup into a separate `html` file and use it in the iframe. 
Alse, we try not to use jquery in the tests.